components/tribe/searchMembers.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FunctionComponent } from "react";
+import { useEffect } from "react";
+import { useState } from "react";
+import { cleanUsername } from "../../utils/stringService";
+import InputAction from "../UI/inputAction";
+import Member from "./member";
+import styles from "../../styles/jointhetribe.module.css";
+import MembersSkeleton from "./membersSkeletons";
+
+const SearchMembers: FunctionComponent = () => {
+  const [username, setUsername] = useState<string>("");
+  const [loadingState, setLoadingState] = useState(false);
+  const [error, setError] = useState();
+  const [followingTribeMembers, setFollowingTribeMembers] = useState<
+    Member[] | undefined
+  >();
+  //
+
+  function onSearch(username: string): void {
+    setLoadingState(true);
+    setUsername(cleanUsername(username));
+  }
+
+  useEffect(() => {
+    if (username) {
+      fetch(`/api/twitter/get_user_following?username=${username}`)
+        .then((response) => response.json())
+        .then((data) => {
+          setFollowingTribeMembers(data);
+          setLoadingState(false);
+        })
+        .catch((error) => {
+          console.error("error", error);
+          setError(error);
+          setLoadingState(false);
+        });
+    }
+  }, [username]);
+
+  return (
+    <div
+      id="me-and-the-tribe"
+      className="flex justify-center flex-col items-center"
+    >
+      <InputAction
+        placeholder="@Twitter username"
```

Comment:
![image](https://user-images.githubusercontent.com/41830259/224560450-4576f281-e326-4e17-a28d-da56f74656a7.png)

text is a bit too big